### ISSUE TITLE
Video Controls Missing

### ISSUE THREAD

**Issue by RyanGreenup** [gollum/gollum#1856]
**Describe the bug**
When embedding a video the controls are removed from the HTML.

**To Reproduce**
  1. Install Gollum with `gem install gollum`
  2. Add a video to a page 
    ```
    <<Video(media/my_video.webm)>>
    ```
  3. Inspect the HTML through the browser, note that the `control=""` part of the macro has been removed.

**Expected behavior**
It is expected that the output html would have `control=""` so that the rendered video can be played. This is a bigger issue on *Android*, where there is no way to enable the controls through the browser.
** What I've Tried **
Adding the following to `config.rb`, didn't help (as suggested in #1587 ):

```ruby
::Loofah::HTML5::SafeList::ACCEPTABLE_ATTRIBUTES.add('controls')
::Loofah::HTML5::SafeList::ALLOWED_ATTRIBUTES.add('controls')
```
To validate the above, I can confirm, the following did indeed disable the video element:

```ruby
::Loofah::HTML5::SafeList::ALLOWED_ELEMENTS_WITH_LIBXML2.delete('video')
```

**Screenshots**
<details>

![image](https://user-images.githubusercontent.com/47832594/189576076-e66c3184-87af-48b4-8b9f-713326dcca99.png)

</details>

<details>

![image](https://user-images.githubusercontent.com/47832594/189576121-f4d1dd06-2258-411a-8df0-ffd48dc40a36.png)

<details>

**Environment Info**

<details>
Gollum 5.3.0
Running on: x86_64-linux with Ruby version 3.0.4
Using:
rugged 1.1.1
gollum-rugged_adapter 1.1.2
gollum-lib 5.2
Markdown rendering gem: pandoc-ruby
Other renderering gems:
org-ruby 0.9.12

</details>

**Comment by dometto** [comment]
The issue seems to be with the fact that the `controls` attribute is set to `=""` by the `Video()` Macro, which is causing Loofah to remove the attribute even if it is added to the `SafeList`. This seems to be a bug in Loofah (https://github.com/flavorjones/loofah/issues/242), unless there is some reason for this behavior that I don't understand.

A workaround is to use the following HTML:

```html
<video  width="100%" height="100%" src="my_video.webm" controls="true">
```

We could fix the Macro by setting `controls` to `true`, but if it's indeed a bug in Loofah that shouldn't be necessary.

**Comment by dometto** [comment]
`gem update gollum-lib` to fix this issue (fixed in `gollum-lib v5.2.1`)
